github/pulls_reviewers_test.go

```diff
@@ -13,6 +13,110 @@ import (
 	"testing"
 )
 
+func TestReviewersRequest_marshall(t *testing.T) {
+	testJSONMarshal(t, &ReviewersRequest{}, "{}")
+
+	u := &ReviewersRequest{
+		NodeID:        String("n"),
+		Reviewers:     []string{"r"},
+		TeamReviewers: []string{"t"},
+	}
+
+	want := `{
+		"node_id": "n",
+		"reviewers": [
+			"r"
+		],
+		"team_reviewers" : [
+			"t"
+		]
+	}`
+
+	testJSONMarshal(t, u, want)
+}
+
+func TestReviewers_marshall(t *testing.T) {
```

Comment:
Same here... can you please change this from `marshall` to `marshal` ?

Reply:
Made the change